app/mixins/render-component.js

```diff
@@ -0,0 +1,12 @@
+import {on} from '@ember/object/evented';
+import Mixin from '@ember/object/mixin';
+
+export default Mixin.create({
+	replacePlaceholder: on('didInsertElement', function () {
```

Comment:
please consider adding _
replacePlaceholder might be used by a component
and if it is - it might lead to hard to find bugs

same goes for placeholderElement